newIDE/app/.storybook/app-level-styling.css

```diff
@@ -0,0 +1,22 @@
+@keyframes text-focus-in {
+  0% {
+    -webkit-filter: blur(12px);
+    filter: blur(12px);
+    opacity: 0;
+  }
+
+  100% {
+    -webkit-filter: blur(0px);
+    filter: blur(0px);
+    opacity: 1;
+  }
+}
+
+@keyframes verticalEaseShake {
```

Comment:
Any chance we can use kebab-case or camelCase for both?

Reply:
Might be better to move this/import in newIDE/app/src/UI/Theme/Global/Animation.css